marimo/_ast/names.py

```diff
@@ -0,0 +1,13 @@
+import hashlib
+
+DEFAULT_CELL_NAME = "__"
+
+
+def is_default_cell_name(name: str) -> bool:
+    return name.startswith("__") and (name == "__" or len(name) == 6)
```

Comment:
Instead of `is_default_cell_name`, should this be `is_internal_cell_name` with implementation

```python
return name.startswith(DEFAULT_CELL_NAME)
```

That is, let's just treat cell names starting with `__` as "reserved" cell names. I doubt people are using custom names starting with `__` today anyway.

I think that will make migrations easier should we decide to change the representation of the default cell name in the future.

Reply:
sounds good